client/src/app/pages/vulnerability-details/overview.tsx

```diff
@@ -24,76 +19,52 @@ interface OverviewProps {
-                      </DescriptionListDescription>
-                    </DescriptionListGroup>
-                    <DescriptionListGroup>
-                      <DescriptionListTerm>Withdrawn</DescriptionListTerm>
-                      <DescriptionListDescription>
-                        {formatDate(vulnerability.withdrawn)}
-                      </DescriptionListDescription>
-                    </DescriptionListGroup>
-                  </DescriptionList>
-                </CardBody>
-              </Card>
-            </GridItem>
-            <GridItem md={4}>
-              <Card isFullHeight>
-                <CardTitle>Additional info</CardTitle>
-                <CardBody>
-                  <DescriptionList>
-                    <DescriptionListGroup>
-                      <DescriptionListTerm>CWE</DescriptionListTerm>
-                      <DescriptionListDescription>
-                        {"vulnerability.cwe"}
-                      </DescriptionListDescription>
-                    </DescriptionListGroup>
-                    <DescriptionListGroup>
-                      <DescriptionListTerm>Non normative</DescriptionListTerm>
-                      <DescriptionListDescription>
-                        {vulnerability.normative ? "Yes" : "No"}
-                      </DescriptionListDescription>
-                    </DescriptionListGroup>
-                  </DescriptionList>
-                </CardBody>
-              </Card>
-            </GridItem>
-          </Grid>
-        </StackItem>
-      </Stack>
+      <TextContent>
+        <Text>{vulnerability.title || vulnerability.description}</Text>
+      </TextContent>
+      <br />
+      <DescriptionList
+        columnModifier={{
+          default: "2Col",
+        }}
+      >
+        <DescriptionListGroup>
+          <DescriptionListTerm>Published</DescriptionListTerm>
+          <DescriptionListDescription>
+            {formatDate(vulnerability.published)}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>Modified</DescriptionListTerm>
+          <DescriptionListDescription>
+            {formatDate(vulnerability.modified)}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>Released</DescriptionListTerm>
+          <DescriptionListDescription>
+            {formatDate(vulnerability.released)}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>Withdrawn</DescriptionListTerm>
+          <DescriptionListDescription>
+            {formatDate(vulnerability.withdrawn)}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>CWE</DescriptionListTerm>
+          <DescriptionListDescription>
+            {"vulnerability.cwe"}
```

Comment:
I think the property is `cwes`. And this is a mistake in the current main branch as it is rendering a string and not the value of the variable